### ISSUE TITLE
latexmk error

### ISSUE THREAD

**Issue by shibukawa** [sphinx-doc/sphinx#3543]
Subject: latexmk error

### Problem
- Fail creating PDF via latexpdf command on master branch

#### Procedure to reproduce the problem

I added CI setting of creating PDF. Summary is:

* Install Ubuntu 16.04LTS
* Install texlive texlive-latex-extra texlive-lang-cjk python-pip make via apt.
* Install Sphinx from master branch.

* Last succeeded build was: March 5th 17:43 UTC
* Build failed when March 10th 23:57 UTC

#### Error logs / results

```
make[1]: latexmk: Command not found
Makefile:33: recipe for target 'RealWorldHttp.pdf' failed
make[1]: Leaving directory '/builds/shibukawa/httpbook/_build/latex'
make[1]: *** [RealWorldHttp.pdf] Error 127
Makefile:146: recipe for target 'latexpdf' failed
make: *** [latexpdf] Error 2
ERROR: Build failed: exit code 1
```

[log.txt](https://github.com/sphinx-doc/sphinx/files/835768/log.txt)

#### Expected results
Successfully creating PDF

### Reproducible project / your project
- <link to your project, or attach zipped small project sample>

### Environment info
- OS: Ubuntu 16.04LTS on GitLab CI
- Python version: Python 2.7
- Sphinx version: master branch

Build script

```yaml
image: chezou/sphinx-recommonmark:latest

pdf:
  script:
    - apt-get udpate & apt-get install -y -q git
    - pip install git+https://github.com/sphinx-doc/sphinx
    - pip install sphinxcontrib-blockdiag
    - pip install sphinxcontrib-seqdiag
    - make latexpdf
  artifacts:
    paths:
    - _build/latex/mybook.pdf
  only:
    - master
```

Dockerfile (chezou/sphinx-recommonmark)

https://github.com/chezou/docker-sphinx-recommonmark/blob/master/Dockerfile

**Comment by tk0miya** [comment]
Since #3082, we use `latexmk` command to build PDF.
so you need to install latexmk package.
http://packages.ubuntu.com/xenial/latexmk

**Comment by shibukawa** [comment]
Thank you

**Comment by jfbu** [comment]
The merged PR #3082 contained an update to the builders docs (19558c9), but I had not yet updated CHANGES, now done at 538f55e. By the way, perhaps the info at 19558c9 should be moved to location more visible in Sphinx docs ?

**Comment by tk0miya** [comment]
Okay, let's add "dependency" section to CHANGES file.
